ab2d-filters/src/main/java/gov/cms/ab2d/filter/ExplanationOfBenefitTrimmerR4.java

```diff
@@ -170,7 +170,9 @@ private static ExplanationOfBenefit copyData(ExplanationOfBenefit benefit) {
         copy.setLanguage(benefit.getLanguage());
         copy.setImplicitRules(benefit.getImplicitRules());
 
-        copy.setExtension(benefit.getExtensionsByUrl(NL_RECORD_IDENTIFICATION));
+        List<Extension> extensions = new ArrayList<>();
+        extensions.addAll(benefit.getExtensionsByUrl(NL_RECORD_IDENTIFICATION));
```

Comment:
Don't really need the addAll(), just pass the collection to the constructor.  This avoids re-sizing the list.